ddsrouter_test/compose/test_cases/frequency/max-tx-rate/compose.yml

```diff
@@ -0,0 +1,63 @@
+# Test description:
```

Comment:
Test not being executed (yaml file is also missing)

Reply:
I forgot to delete that compose.yaml. Woops.